test/items/double-battle-chance-booster.test.ts

```diff
@@ -71,7 +71,7 @@ describe("Items - Double Battle Chance Boosters", () => {
       "SelectModifierPhase",
       UiMode.MODIFIER_SELECT,
       () => {
-        const handler = game.scene.ui.getHandler() as ModifierSelectUiHandler;
+        const handler = game.scene.ui.getCurrentHandler() as ModifierSelectUiHandler;
```

Comment:
NIT: Use generic param `<T>`

```suggestion
        const handler = game.scene.ui.getCurrentHandler<ModifierSelectUiHandler>();
```